Makefile

```diff
@@ -4,34 +4,47 @@ build-prod:
 build-dev:
 	docker build -t openslides-auth-dev -f Dockerfile.dev .
 
-run-bash: | build-dev
-	docker-compose -f docker-compose.dev.yml up -d
-	docker-compose -f docker-compose.dev.yml exec auth wait-for-it auth:9004
-	docker-compose -f docker-compose.dev.yml exec auth bash
-	docker-compose -f docker-compose.dev.yml down
+build-test:
+	docker build -t openslides-auth-dev -f Dockerfile.test .
 
-run-check-cleanup: | build-dev
-	docker-compose -f docker-compose.dev.yml up -d
-	docker-compose -f docker-compose.dev.yml exec auth wait-for-it auth:9004
-	docker-compose -f docker-compose.dev.yml exec auth npm run lint-check
-	docker-compose -f docker-compose.dev.yml exec auth npm run prettify-check
-	docker-compose -f docker-compose.dev.yml down
+run-dev: | build-dev
+	docker-compose -f docker-compose.dev.yml up
+	stop-dev
 
-run-cleanup: | build-dev
+run-pre-test: | build-test
 	docker-compose -f docker-compose.dev.yml up -d
-	docker-compose -f docker-compose.dev.yml exec auth wait-for-it auth:9004
-	docker-compose -f docker-compose.dev.yml exec auth npm run cleanup
+	docker-compose -f docker-compose.dev.yml exec -T auth ./wait-for.sh auth:9004
+
+run-bash: | run-pre-test
+	docker-compose -f docker-compose.dev.yml exec auth sh
 	docker-compose -f docker-compose.dev.yml down
 
-run-tests: | build-dev
+run-check-lint: | run-pre-test
+	docker-compose -f docker-compose.dev.yml exec -T auth npm run lint-check
+
+run-check-prettify: | run-pre-test
+	docker-compose -f docker-compose.dev.yml exec -T auth npm run prettify-check
+
+run-test: | run-pre-test
 	@echo "########################################################################"
 	@echo "###################### Start full system tests #########################"
 	@echo "########################################################################"
-	docker-compose -f docker-compose.dev.yml up -d
-	docker-compose -f docker-compose.dev.yml exec -T auth wait-for-it auth:9004
 	docker-compose -f docker-compose.dev.yml exec -T auth npm run test
 	docker-compose -f docker-compose.dev.yml exec -T auth pytest
+
+run-cleanup: | build-dev
+	docker-compose -f docker-compose.dev.yml up -d
+	docker-compose -f docker-compose.dev.yml exec auth ./wait-for.sh auth:9004
+	docker-compose -f docker-compose.dev.yml exec auth npm run cleanup
 	docker-compose -f docker-compose.dev.yml down
 
-stop-run:
+run-test-and-stop: | run-test
+	stop-dev
+
+run-prod: | build-prod
```

Comment:
more like test-prod since you are stopping it again.

The location of the docker-compose.yml file in the root is very prominent since it is only used in the CI; maybe move it away, so nobody wants to really start it this way (In https://github.com/OpenSlides/openslides-backend/pull/897 I creted an extra folder `.github/startup_test`).